project_euler/problem_5/sol5.dart

```diff
@@ -0,0 +1,25 @@
+// Author : Devmaufh
+// Email : mau1361317@gmail.com
+
+/**
+ * [Problem 5](https://projecteuler.net/problem=5) solution
+ * Problem Statement:
+ * 2520 is the smallest number that can be divided by each of the numbers from 1 to 10 without any remainder.
+ * What is the smallest positive number that is evenly divisible by all of the numbers from 1 to 20?
+ */
+
+void main(){
+  int number = 21;
+  LOOP: while(true){
+    bool isSolution = true;
```

Comment:
You could just parametrize this while loop with `isSolution`
```suggestion
  bool isSolution = false;
  while(!isSolution){
    isSolution = true;
    number++;
```